<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndImportRadar</class>
 <widget class="QDialog" name="wndImportRadar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Define Radar Id constraints</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="cbSetFixedPort">
     <property name="text">
      <string>Fixed Serial Port (different modules can be attached to the same serial port)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cbSetFixedRadarId">
     <property name="text">
      <string>Fixed Radar Id (the same module can be attached to different serial ports)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnOk">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
